packages/storage-ui/src/Components/Modules/AddCIDModal.tsx

```diff
@@ -80,17 +82,19 @@ const AddCIDModal = ({ modalOpen = false, close }: IAddCIDModuleProps) => {
   const [cidError, setCidError] = useState("")
   const [cid, setCid] = useState("")
   const [showWarning, setShowWarning] = useState(false)
+  const [name, setName] = useState("")
 
   const onClose = useCallback(() => {
     setCid("")
     setShowWarning(false)
     setCidError("")
+    setName("")
     close()
   }, [close])
 
   const onSubmit = useCallback(() => {
     setAccessingCID(true)
-    addPin(cid)
+    addPin(cid, name)
```

Comment:
Improvement, prevents submitting when there's no CID in the input.
```suggestion
    if (!cid) return
    addPin(cid, name)
```

Reply:
The button is disabled, so it shouldn't be possible. But it can't hurt.